lib/matplotlib/backends/__init__.py

```diff
@@ -26,30 +29,31 @@ def pylab_setup(backend=None):
         The module which contains the backend of choice
 
     new_figure_manager : function
-        Create a new figure manage (roughly maps to GUI window)
+        Create a new figure manager (roughly maps to GUI window)
 
     draw_if_interactive : function
         Redraw the current figure if pyplot is interactive
 
     show : function
-        Show (and possible block) any unshown figures.
+        Show (and possibly block) any unshown figures.
 
     '''
     # Import the requested backend into a generic module object
-    if backend is None:
-        backend = matplotlib.get_backend()  # validates, to match all_backends
```

Comment:
calling `matplotlib.get_backend()` in this function was the main point of the PR that broke this.  In either case (passed in arg or calling `get_backend`) the value just needs to be mirrored out _to_ `backend`.  If we only needed to read this value we would not need `global` as the function would correctly close over it.